templates/adherent/profile.html.twig

```diff
@@ -82,6 +82,13 @@
         <div class="space--10-0"></div>
 
         <div class="l__row l__row--top l__row--h-stretch l__mobile--col">
+
+            <div class="l__row l__row--h-stretch b__nudge--bottom-tiny form__radio-button">
+                {{ form_widget(form.nationality, { attr: { class: 'register__form__profession' } }) }}
```

Comment:
why do you need `register__form__profession` class?

Reply:
edited